### PR TITLE
[CLOB-1037] grpc to get all perpetuals and liquidity tiers

### PR DESCRIPTION
**Pull request by jayy04**
### Changelist
[Describe or list the changes made in this PR]

### Test Plan
[Describe how this PR was tested (if applicable)]

### Author/Reviewer Checklist
- [ ] If this PR has changes that result in a different app state given the same prior state and transaction list, manually add the `state-breaking` label.
- [ ] If the PR has breaking postgres changes to the indexer add the `indexer-postgres-breaking` label.
- [ ] If this PR isn't state-breaking but has changes that modify behavior in `PrepareProposal` or `ProcessProposal`, manually add the label `proposal-breaking`.
- [ ] If this PR is one of many that implement a specific feature, manually label them all `feature:[feature-name]`.
- [ ] If you wish to for mergify-bot to automatically create a PR to backport your change to a release branch, manually add the label `backport/[branch-name]`.
- [ ] Manually add any of the following labels: `refactor`, `chore`, `bug`.

### CODE REVIEW

File: protocol/daemons/liquidation/client/grpc_helper.go [hunk 1]
```diff
@@ -190,6 +262,20 @@ func (c *Client) SendLiquidatableSubaccountIds(
 	return nil
 }
 
+// nolint:unused
+func newContextWithQueryBlockHeight(
```
Comment (jayy04) [inline protocol/daemons/liquidation/client/grpc_helper.go:266]: will call this outside of these grpc queries in the main daemon loop. unused for now so silencing lint

File: protocol/daemons/liquidation/client/grpc_helper.go [hunk 2]
```diff
@@ -41,6 +42,86 @@ func (c *Client) GetPreviousBlockInfo(
...
+	liquidityTiers []perptypes.LiquidityTier,
+	err error,
+) {
+	defer metrics.ModuleMeasureSince(
+		metrics.LiquidationDaemon,
+		metrics.DaemonGetAllLiquidityTiersLatency,
+		time.Now(),
+	)
+
+	liquidityTiers = make([]perptypes.LiquidityTier, 0)
```
Comment (jonfung-dydx) [inline protocol/daemons/liquidation/client/grpc_helper.go:100]: dont need to change but i think this is already initialized in  line 91

**Review by jonfung-dydx — APPROVED** [review]

File: protocol/daemons/liquidation/client/grpc_helper.go (hunk 1 above)
Comment (jonfung-dydx) [reply protocol/daemons/liquidation/client/grpc_helper.go:266]: can do in future pr so you don't have to wait for CI to run again

Comment (jayy04) [reply protocol/daemons/liquidation/client/grpc_helper.go:266]: yeah will do in a future PR

**Pull request merged**